src/ScanItems.js

```diff
@@ -108,7 +109,8 @@ class ScanItems extends React.Component {
 
   async fetchItem(barcode) {
     const { mutator } = this.props;
-    const query = `barcode==${barcode}`;
+    const bcode = '"' + escapeCqlValue(barcode) + '"';
+    const query = `barcode==${bcode}`;
```

Comment:
You can stick anything, including an expression, inside `${}` so these lines can combine to
```
    const query = `barcode==${escapeCqlValue(barcode)}`;
```
but what you have is just fine. Fer sure folks will argue which is more clear because folks will argue about _anything_.

Reply:
I tried similar syntax in the checkin module (where I was fixing basically the same problem) and eslint did not like it-the error message was something to the effect of "I don't want curly brace expressions in string literals."